entries/b/bank.lendingclub.com.json

```diff
@@ -0,0 +1,23 @@
+{
+  "LendingClub Bank": {
+    "domain": "bank.lendingclub.com",
+    "img": "lendingclub.com.svg",
+    "additional-domains": [
+      "banking.lendingclub.com",
+      "radiusbank.com"
+    ],
+    "tfa": [
+      "sms",
+      "call",
+      "email",
+      "totp"
+    ],
```

Comment:
The doc page doesn't list the methods unfortunately, and on https://radiusbank.custhelp.com/app/answers/detail/a_id/70 they mention "email or text" only. Do you have further information?

Reply:
I got in touch with support and they confirmed that SMS, call and email are supported

> Yes, we offer two factor authentication to be received via emails and for SMS/phone call.